src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -32,121 +32,129 @@ the main body of the message as well as a quote.
+						:aria-label="sendingErrorIconTooltip"
+						tabindex="0"
+						@mouseover="showReloadButton = true"
+						@focus="showReloadButton = true"
+						@mouseleave="showReloadButton = true"
+						@blur="showReloadButton = true">
+						<button
+							v-if="sendingErrorCanRetry && showReloadButton"
+							class="nc-button nc-button__main--dark"
+							@click="handleRetry">
+							<Reload
+								decorative
+								title=""
+								:size="16" />
+						</button>
+						<AlertCircle v-else
+							decorative
+							title=""
+							:size="16" />
+					</div>
+					<div v-else-if="isTemporary && !isTemporaryUpload || isDeleting"
+						v-tooltip.auto="loadingIconTooltip"
+						class="icon-loading-small message-status"
+						:aria-label="loadingIconTooltip" />
+					<div v-else-if="showCommonReadIcon"
+						v-tooltip.auto="commonReadIconTooltip"
+						class="message-status"
+						:aria-label="commonReadIconTooltip">
+						<CheckAll decorative
+							title=""
+							:size="16" />
+					</div>
+					<div v-else-if="showSentIcon"
+						v-tooltip.auto="sentIconTooltip"
+						class="message-status"
+						:aria-label="sentIconTooltip">
+						<Check decorative
+							title=""
+							:size="16" />
+					</div>
+					<!-- Message Actions -->
+					<Actions
+						v-show="showActions && hasActions"
+						class="message__main__right__actions"
+						:class="{ 'tall' : isTallEnough }">
+						<ActionButton
+							v-if="isReplyable"
+							icon="icon-reply"
+							:close-after-click="true"
+							@click.stop="handleReply">
+							{{ t('spreed', 'Reply') }}
+						</ActionButton>
 						<ActionButton
-							:key="action.label"
-							:icon="action.icon"
+							v-if="isPrivateReplyable"
+							icon="icon-user"
 							:close-after-click="true"
-							@click="action.callback(messageAPIData)">
-							{{ action.label }}
+							@click.stop="handlePrivateReply">
+							{{ t('spreed', 'Reply privately') }}
 						</ActionButton>
-					</template>
-					<ActionButton
-						v-if="isDeleteable"
-						icon="icon-delete"
-						:close-after-click="true"
-						@click.stop="handleDelete">
-						{{ t('spreed', 'Delete') }}
-					</ActionButton>
-				</Actions>
+						<template
+							v-for="action in messageActions">
+							<ActionButton
+								:key="action.label"
+								:icon="action.icon"
+								:close-after-click="true"
+								@click="action.callback(messageAPIData)">
+								{{ action.label }}
+							</ActionButton>
+						</template>
+						<ActionButton
+							v-if="isDeleteable"
+							icon="icon-delete"
+							:close-after-click="true"
+							@click.stop="handleDelete">
+							{{ t('spreed', 'Delete') }}
+						</ActionButton>
+					</Actions>
+				</div>
+			</div>
+			<div v-if="isLastReadMessage"
+				v-observe-visibility="lastReadMessageVisibilityChanged">
+				<div v-if="!isLastMessage" class="new-message-marker">
+					<span>----- {{ t('spreed', 'Unread messages') }} -----</span>
```

Comment:
let's use the old date time styling instead of ascii, see removed code in https://github.com/nextcloud/spreed/pull/4582/files

Reply:
it would look something like this:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/277525/109502677-9860d580-7a99-11eb-85ba-f817260512ab.png">

and would obsolete your comment about the background color below

but if we make it "floating" instead of "in-layout" we probably can't use the non-bubble style as it wouldn't overlap well on top of other messages